tox.ini

```diff
@@ -24,41 +24,13 @@ commands =
     python -m pip list
     python test.py -v
 
-# Env to create coverage report locally
-[testenv:localcoverage]
```

Comment:
Removed since this does not exist in hdmf and we can look at the report on codecov itself.